test/suites/standard/hostname.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Documentation       Tests verifying hostname resolution
+
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/oc.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           restart    slow
+
+
+*** Variables ***
+${NEW_HOSTNAME}     microshift.local
+${OLD_HOSTNAME}     ${EMPTY}
+
+
+*** Test Cases ***
+Verify local name resolution
+    [Documentation]    Verify correct name resolution through mDNS
+    [Setup]    Configure New Hostname
+
+    Oc Wait For    deployment/router-default    condition\=Available    120s    ns=openshift-ingress
+    Oc Logs    namespace="openshift-ingress"    opts="deployment/router-default"
```

Comment:
Is this just for debugging?

Reply:
Or, we should be expecting a certain output?

Reply:
The actual bug consists in not being able to retrieve logs from a host configured with `.local` suffix hostname.
When issuing a request to the apiserver asking for logs, it will go check the node IP using the name and thats when mDNS resolution needs to happen.
We dont really care about the logs contents, we care about being able to retrieve them (any container, actually).